client/src/ui/BitbakeCommands.ts

```diff
@@ -215,7 +215,7 @@ async function selectRecipe (bitbakeWorkspace: BitbakeWorkspace, bitBakeProjectS
       bitbakeWorkspace.activeRecipes,
       devtoolWorkspacesNames)
 
-    if (canAdd) {
+    if (canAdd || bitbakeWorkspace.activeRecipes.length === 0) {
```

Comment:
Yes, otherwise, the quick pick had no options in that case.
This can happen if the user cancelled the initial scan or it failed. We still don't want to block him from any commands.
I disabled canAdd for the recipe scan because it would create too many entries in the recipes view (each time a different file is saved).

Reply:
I see, not a blocker for me.